modules/statistics/AvgRTPStatsReporter.js

```diff
@@ -452,8 +565,6 @@ export default class AvgRTPStatsReporter {
         this._avgRemoteFPS.reset();
         this._avgLocalFPS.reset();
         this._avgCQ.reset();
-        this._avgRTT.reset();
-        this._avgRemoteRTTMap.clear();
```

Comment:
This is fixed in a later commit (and the answer to my question appears to be "no").